### PR TITLE
Enhance DSL rules for say and interpret (resubmitted)

### PR DESCRIPTION
**Pull request by ThomDietrich**
Submitted by Laurent Garnier <lg.hc@free.fr> (github:lolodomo)
Submitted at https://github.com/openhab/openhab-docs/pull/149

Signed-off-by: Thomas Dietrich <Thomas.Dietrich@tu-ilmenau.de>

### CODE REVIEW

**Comment by ThomDietrich** [comment]
@lolodomo 
Closes https://github.com/openhab/openhab-docs/pull/149

File: configuration/multimedia.md [hunk 1]
```diff
@@ -64,24 +64,27 @@ mactts:Kanya Kanya (th_TH)
...
-
-```
+```java
 say("Hello world!")
+say("Hello world!", "voicerss:enGB")
+say("Hello world!", "voicerss:enUS", "sonos:PLAY5:kitchen")
 ```
 
-within DSL rules.
+The default voice and default audio sink will be used.
```
Comment (lolodomo) [inline configuration/multimedia.md]: This sentence only applies to the console commande while the next ones only apply to the DSL rules (meaning you cannot specify a voice or an audio sink with the console command).
The way you rearrange things compared to my initial commit makes this point less clear than before.

Comment (ThomDietrich) [inline configuration/multimedia.md]: Oh :disappointed: I'm sorry, besides Splitting lines, that's probably the only thing I "improved". My fault, will change back to the way you originally phrased it.

**Comment by ThomDietrich** [comment]
@lolodomo it's corrected. One more general concern: Just mentioning "Use say("blaaa")" seems a bit ambiguous. A user might want to learn about the details of this function. Would linking to the correct javadoc be wise? I suspect that to be https://www.eclipse.org/smarthome/documentation/javadoc/org/eclipse/smarthome/core/voice/VoiceManager.html#say-java.lang.String-

Looking for playSound was not as easy. It seems to be implemented here but something is fishy about that... https://github.com/eclipse/smarthome/blob/d1a84433089f7e7e1d9be5a47f3d36f5cd4053a4/bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/Audio.java

**Comment by lolodomo** [comment]
For DSL rules, code for say and interpret are in this file: https://github.com/eclipse/smarthome/blob/master/bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/Voice.java.

**Comment by lolodomo** [comment]
Be careful, for playSound, you should rather use this link: https://github.com/eclipse/smarthome/blob/master/bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/Audio.java

**Comment by ThomDietrich** [comment]
My link was pointing to javadocs, which should be the preferred resource. However there are some discrepancies and indeed, it was last generated "Tue Sep 27 15:56:41 CEST 2016", that's before the commit linked above. Well :hankey: I'll have to file an issue for that.

**Comment by lolodomo** [comment]
If you think there are more details in code comments, you can copy from javadoc.

**Comment by ThomDietrich** [comment]
Well there should ^^ 
In the meantime the javadoc is back up to date. I'll add a few links later. Thanks for your help

**Pull request merged**